contracts/src/instances/eeth/EETHConversions.sol

```diff
@@ -0,0 +1,71 @@
+// SPDX-License-Identifier: Apache-2.0
+pragma solidity 0.8.20;
+
+import { IERC20 } from "../../interfaces/IERC20.sol";
+import { ILiquidityPool } from "../..//interfaces/ILiquidityPool.sol";
+import { FixedPointMath, ONE } from "../../libraries/FixedPointMath.sol";
+import { IEETH } from "../..//interfaces/IEETH.sol";
```

Comment:
Nit:

```suggestion
import { IEETH } from "../../interfaces/IEETH.sol";
```

I think we need to resort these imports. This should go above the `libraries` import.